Dockerfile

```diff
@@ -96,6 +96,8 @@ RUN apt-get -qq -y update \
     tesseract-ocr-aze \
     tesseract-ocr-bel \
     tesseract-ocr-uzb \
+    ### pdf convert \
+    libreoffice \
```

Comment:
Do you need to install a bunch of font packages to make this generate reasonable-looking PDFs? There was quite a lot of junk installed in the old container, most of which came about from broken PDFs being generated.

Reply:
Oh, I tried the minimum footprint approach, but I will need to do some more testing then. Thanks!